src/styles/blocks/page.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: "Roboto" ;
+  src: url(../fonts/Roboto/Roboto-Regular.ttf);
+}
+
+.page-content {
+  font-family: "Roboto", sans-serif;
+  padding: 10px;
+  box-sizing: border-box;
```

Comment:
box-sizing may be moved to * selector
But only box-sizing, don’t set any other rules there